src/ci/github-actions/jobs.yml

```diff
@@ -0,0 +1,46 @@
+x--expand-yaml-anchors--remove:
```

Comment:
We should add a comment here saying this file does **not** need to anchors to be expanded when you change it. This way, people used to run the command every time they changed the CI config don't run it needlessly.